### DISCUSSION TITLE
Exclusion feature? مستثنی کردن آی‌پی/دامنه/اپ

### DISCUSSION THREAD

**Discussion by aewsrtdfyuiop** [hiddify/hiddify-app#870] [Q&A]
داریم همچین چیزی؟ کجا؟ اگه نه لطفا یه فکری به حالش بشه سپاس.

**Comment by Morrigan-Ship** [comment]
same

**Comment by BaziForYou** [comment]
https://github.com/hiddify/hiddify-next/issues/210
they marked that as Done but still can't find any docs still it's automated rules apply and didn't add this feature if it's so important to you to use V2rayN-Pro for now until they add

**Reply by Morrigan-Ship** [reply]
bacause they think our mean is Region settings.   :)
